game/src/plugins/cmd/src/item-cmd.plugin.kts

```diff
@@ -0,0 +1,24 @@
+import org.apollo.cache.def.ItemDefinition
+import org.apollo.game.model.entity.setting.PrivilegeLevel
+
+on_command("item", PrivilegeLevel.ADMINISTRATOR)
+        .then { player ->
+            if  (!valid_arg_length(arguments, 1..2, player, "Invalid syntax - ::item [id] [amount]")) {
+                return@then

```

Comment:
'return' is not allowed. Followed IDEA's autocomplete and that was the only option